style.css

```diff
@@ -0,0 +1,385 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+@media screen and (max-width: 375px) {
+  body {
+    position: relative;
+    width: 375px;
+    height: 5410px;
+    background: #f7f7f9;
+  }
+
+  .section_1 {
+    background-image: url(images/mobile-hearder_background.jpg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    height: 100%;
+    border-bottom-left-radius: 100px 60px;
+  }
+
+  .container {
+    display: flex;
+    height: 180vh;
+    flex-direction: column;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .container > * {
+    flex: 1 1 auto;
+  }
+
+  .app-header {
+    display: flex;
+    justify-content: space-between;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    padding: 24px;
+  }
+
+  .connect {
+    color: rgba(127, 140, 255, 1);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .social_media1 {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding: 24px;
+  }
+
+  .social_media1 a {
+    padding-right: 10px;
+  }
+
+  .greatting {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    padding: 24px;
+    margin-bottom: -40px;
+  }
+
+  .section_1 p {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    padding: 24px;
+    margin-bottom: -40px;
+  }
+
+  /* Working on second phase */
+  .section_2 {
+    display: grid;
+    grid-template-rows: auto auto auto auto;
+    grid-gap: 10px;
+    align-items: center;
+    padding: 16px;
+    gap: 12px;
+    width: 327px;
+    height: 552px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    margin-top: 130px;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] This is the project card. it should be aligned with Flexbox not grid. 😄 
![inst](https://user-images.githubusercontent.com/99656687/236603464-9473fa37-08df-4255-bff5-20f62c5ad126.png)